src/main/java/org/bonitasoft/web/client/api/AuthenticationApi.java

```diff
@@ -89,19 +63,6 @@ ApiResponse<Void> loginWithHttpInfo(@Param("username") String username, @Param("
     })
     void logout(@Param("redirect") String redirect);
 
-    /**
-     * Logout the current user
-     * Similar to <code>logout</code> but it also returns the http response headers .
-     * Logout the current user from the system
-     * 
-     * @param redirect Setting the redirect parameter to false indicates that the service should not redirect to the login page after logging out. (optional)
-     */
-    @RequestLine("GET /logoutservice?redirect={redirect}")
-    @Headers({
-            "Accept: application/json",
-    })
-    ApiResponse<Void> logoutWithHttpInfo(@Param("redirect") String redirect);
```

Comment:
Why is this method removed?

Reply:
because it is not working. The initial generation created it but it does not take into consideration the fact that the interceptor that set ApiResponse return type does it only for json content type response which is not the case for this method. so it fails with a npe. The method login should be (and is) used instead by BonitaClient